### PR TITLE
Uniform klog output starts with uppercase

### PR DESCRIPTION
**Pull request by Fish-pro**
**What type of PR is this?**

/kind cleanup

**What this PR does / why we need it**:

**Which issue(s) this PR fixes**:
Fixes #

**Special notes for your reviewer**:

In the existing error log, parts begin in lower case，this pr uniform error log output starts with uppercase

**Does this PR introduce a user-facing change?**:

```release-note
NONE
```

### CODE REVIEW

**Comment by Garrybest** [comment]
I like this patch, thanks a lot👍.

File: pkg/util/lifted/pubkeypin/pubkeypin_test.go [hunk 1]
```diff
@@ -160,7 +160,7 @@ func TestHash(t *testing.T) {
 	actualHash := Hash(testCert(t, testCertPEM))
 	if actualHash != expectedHash {
 		t.Errorf(
```
Comment (Garrybest) [inline pkg/util/lifted/pubkeypin/pubkeypin_test.go:162]: I guess we don't need to change test files because the log file does not using klog.

Comment (Fish-pro) [reply pkg/util/lifted/pubkeypin/pubkeypin_test.go]: @Garrybest Thanks for pointing out that it has been revised

**Review by Garrybest — CHANGES_REQUESTED** [review]
I think we still have other codes need to be modify like https://github.com/karmada-io/karmada/blob/eda7814ffc9f69db7f1e066ad24544fc7dcc19a7/cmd/controller-manager/app/controllermanager.go#L167

**Comment by Fish-pro** [comment]
> I think we still have other codes need to be modify like
> 
> https://github.com/karmada-io/karmada/blob/eda7814ffc9f69db7f1e066ad24544fc7dcc19a7/cmd/controller-manager/app/controllermanager.go#L167

@Garrybest Due to the large number of modified files, I did not modify all of them. Now I have searched for modifications globally

**Comment by Garrybest** [comment]
Good job👍.

/lgtm

**Review by RainbowMango — APPROVED** [review]
/approve

**Pull request merged**